### PR TITLE
Fix gl_VertexID value in base-vertex-base-instance extension test

### PR DESCRIPTION
**Pull request by shrekshao**
> The index of any element transferred to the GL by DrawArraysOneInstance
is referred to as its vertex ID, and may be read by a vertex shader as gl_VertexID.
The vertex ID of the ith element transferred is first + i.

From the [OpenGL ES spec section 10.5](https://www.khronos.org/registry/OpenGL/specs/es/3.2/es_spec_3.2.pdf)

Related to #3278

### CODE REVIEW

**Comment by shrekshao** [comment]
@jdashg does this change look good to you?

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html [hunk 1]
```diff
@@ -310,7 +310,7 @@
       const vert_src = `\
 #version 300 es
 #line 405
-layout(location = 0) in int a_vertex_id; // Same as first_vert+gl_VertexID
+layout(location = 0) in int a_vertex_id; // COUNT_UP_DATA
```
Comment (jdashg) [inline sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:313]: "Same as gl_VertexID". I'm not sure what COUNT_UP_DATA is

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html [hunk 2]
```diff
@@ -469,8 +469,8 @@
     gl.disable(gl.SCISSOR_TEST);
     gl.clearBufferfv(gl.COLOR, 0, [1,0,0,1]);
 
-    const last_gl_vert_id = desc.base_vert + desc.vert_count - 1;
-    const last_vert_id = desc.first_vert + last_gl_vert_id;
+    const last_gl_vert_id = desc.first_vert + desc.base_vert + desc.vert_count - 1;
```
Comment (jdashg) [inline sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:472]: `desc.base_vert + desc.first_vert`

**Review by jdashg — CHANGES_REQUESTED** [review]

**Comment by jdashg** [comment]
Oh, and *add that spec reference to the test*!!

File: sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html (hunk 2 above)
Comment (shrekshao) [reply sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:472]: vert_count can be 0, 1, 2 so they are not equal?

Comment (shrekshao) [reply sdk/tests/conformance2/extensions/webgl-multi-draw-instanced-base-vertex-base-instance.html:472]: Oh I think you mean flip the order...
```diff
- const last_gl_vert_id = desc.first_vert + desc.base_vert + desc.vert_count - 1;
+ const last_gl_vert_id = desc.base_vert + desc.first_vert + desc.vert_count - 1;
```

**Review by jdashg — APPROVED** [review]

**Comment by jdashg** [comment]
Thanks!

**Pull request merged**